### PR TITLE
feat: Kubernetes storage integration tests doc

### PR DESCRIPTION
**Pull request by nabokihms**
Related https://github.com/dexidp/dex/pull/2091

### CODE REVIEW

File: content/docs/development/integration-tests.md [hunk 1]
```diff
@@ -63,6 +63,30 @@ These tests can also be executed using docker:
   ```
 - clean up the etcd container: `docker rm -f dex-etcd`
 
+## Kubernetes
+
+Running integration tests for Kubernetes storage require the `DEX_KUBERNETES_CONFIG_PATH` environment variable
+with the path to kubeconfig file of the existing cluster. For tests, it is ok to use "mini" Kubernetes distributive, e.g., [KinD][kind], [Microk8s][microk8s].
```
Comment (nate-double-u) [inline content/docs/development/integration-tests.md:69]: I'm not sure I understand this first sentence exactly. I think this fixes it, but am not sure if it is exactly true as per the actual config requirements:

```suggestion
Running integration tests for Kubernetes storage requires the `DEX_KUBERNETES_CONFIG_PATH` environment variable
be set with the path to kubeconfig file of the existing cluster. For tests, it is ok to use "mini" Kubernetes distributive, e.g., [KinD][kind], [Microk8s][microk8s].
```

File: content/docs/development/integration-tests.md [hunk 2]
```diff
@@ -63,6 +63,30 @@ These tests can also be executed using docker:
...
+
+There is an example for running tests in the KinD cluster.
```
Comment (nate-double-u) [inline content/docs/development/integration-tests.md:71]:
```suggestion
Example KinD cluster test run:
```

File: content/docs/development/integration-tests.md [hunk 3]
```diff
@@ -63,6 +63,30 @@ These tests can also be executed using docker:
...
+
+* Install KinD using the instruction from the [official website][kind-install].
```
Comment (nate-double-u) [inline content/docs/development/integration-tests.md:73]:
```suggestion
* Install KinD using the instructions from the [official website][kind-install].
```

**Review by nate-double-u — COMMENTED** [review]
I did a quick copy-edit, see comments inline.

**Review by sagikazarmark — APPROVED** [review]

**Pull request merged**